package.json

```diff
@@ -175,7 +175,7 @@
     "node-dir": "0.1.17",
     "node-uuid": "1.4.8",
     "open": "0.0.5",
-    "prettydiff": "1.16.37",
+    "prettydiff2": ">2.2.5",
```

Comment:
Does `prettydiff2` follow semver? If so, this should be `^2.2.5` so no major version is updated.